Gems/GeoJSONSpawner/Code/Source/GeoJSONSpawner/GeoJSONSpawnerUtils.cpp

```diff
@@ -0,0 +1,542 @@
+        distributions.emplace_back(0.0f, stdDevTranslation.GetY());
+        distributions.emplace_back(0.0f, stdDevTranslation.GetZ());
+        distributions.emplace_back(0.0f, stdDevRotation.GetX());
+        distributions.emplace_back(0.0f, stdDevRotation.GetY());
+        distributions.emplace_back(0.0f, stdDevRotation.GetZ());
+        distributions.emplace_back(1.0f, stdDevScale);
+
+        transform.SetTranslation(AZ::Vector3(distributions[0](gen), distributions[1](gen), distributions[2](gen)));
+        const AZ::Quaternion rotation =
+            AZ::Quaternion::CreateFromEulerAnglesDegrees(AZ::Vector3(distributions[3](gen), distributions[4](gen), distributions[5](gen)));
+        transform.SetRotation(rotation);
+        transform.SetUniformScale(AZStd::max(0.0f, distributions[6](gen)));
+        return transform;
+    }
+
+    AZStd::unordered_map<int, AZStd::vector<AzFramework::EntitySpawnTicket>> SpawnEntities(
+        AZStd::vector<GeoJSONSpawnableEntityInfo>& entitiesToSpawn,
+        const AZStd::unordered_map<AZStd::string, GeoJSONSpawnableAssetConfiguration>& spawnableAssetConfigurations,
+        AZ::u64 defaultSeed,
+        const AZStd::string& physicsSceneName,
+        AZ::EntityId parentId)
+    {
+        auto sceneInterface = AZ::Interface<AzPhysics::SceneInterface>::Get();
+        AZ_Assert(sceneInterface, "Unable to get physics scene interface.");
+        const auto sceneHandle = sceneInterface->GetSceneHandle(physicsSceneName);
+
+        AZStd::unordered_map<int, AZStd::vector<AzFramework::EntitySpawnTicket>> groupIdToTicketsMap;
+        auto spawner = AZ::Interface<AzFramework::SpawnableEntitiesDefinition>::Get();
+        AZ_Assert(spawner, "Unable to get spawnable entities definition.");
+
+        for (auto spawnableAsset : spawnableAssetConfigurations)
+        {
+            spawnableAsset.second.m_spawnable.QueueLoad();
+        }
+
+        for (auto& entityToSpawn : entitiesToSpawn)
+        {
+            if (!spawnableAssetConfigurations.contains(entityToSpawn.m_name))
+            {
+                AZ_Error("GeoJSONSpawnerUtils", false, "Spawnable with name [%s] not found.", entityToSpawn.m_name.c_str());
+                continue;
+            }
+
+            const auto& spawnableAssetConfiguration = spawnableAssetConfigurations.at(entityToSpawn.m_name);
+            if (!groupIdToTicketsMap.contains(entityToSpawn.m_id))
+            {
+                groupIdToTicketsMap[entityToSpawn.m_id] = {};
+            }
+
+            for (auto& point : entityToSpawn.m_positions)
+            {
+                AzFramework::SpawnAllEntitiesOptionalArgs optionalArgs;
+                AzFramework::EntitySpawnTicket ticket(spawnableAssetConfiguration.m_spawnable);
+
+                [[maybe_unused]] std::mt19937 gen = std::mt19937(defaultSeed++);
+
+                AZ::Quaternion rotation = AZ::Quaternion::CreateIdentity();
+                AZ::Transform pointTransform{ point, rotation, 1.0f };
```

Comment:
Nitpick, for better visibility: add `float scale = 1.0;` and `AZ::Transform pointTransform{ point, rotation, scale};`